packages/auth/tests/session.spec.ts

```diff
@@ -565,15 +607,16 @@ describe('Wallet integration', function () {
           name: "Test"
         }
       })
-  
-      expect(session.authPromises.length).to.equal(2)
-      await Promise.all(session.authPromises.map((p) => p.promise))
 
-      expect(totalCount).to.equal(2)
-      expect(recoverCount["error"]).to.equal(1)
+      expect((session as any)._allInitialAuthRequests.length).to.equal(networksWithApi.length)
```

Comment:
again, don't like testing to read a private property. super brittle. this'll also break if the session's internal impl changes at all